src/components/CategorySections.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import startCase from 'lodash.startcase';
+
+export const CategorySection = (props) => {
+  const { categoryName, categoryData, styling } = props;
+
+  // if data is an object, not an array
+  if (!Array.isArray(categoryData)) {
```

Comment:
There's a lot of logic going on here. Let me know if there's anything I can do to clarify / if we should do a walkthrough.